src/config/localization/translations.json

```diff
@@ -1792,5 +1792,6 @@
   "Earn %asset%": "Earn %asset%",
   "Credit calculation starts:": "Credit calculation starts:",
   "The start block of the current calculation period. Your average IFO CAKE Pool staking balance is calculated throughout this period.": "The start block of the current calculation period. Your average IFO CAKE Pool staking balance is calculated throughout this period.",
-  "Check out our Medium article for more details.": "Check out our Medium article for more details."
+  "Check out our Medium article for more details.": "Check out our Medium article for more details.",
+  "IFO pool": "IFO pool"
```

Comment:
Change to `IFO Pool` please. Test fails because of it

Reply:
thanks. I really need some sleep